fuzz/fuzz_targets/parse.rs

```diff
@@ -0,0 +1,6 @@
+#![no_main]
+use libfuzzer_sys::fuzz_target;
+
+fuzz_target!(|data: &str| {
+    for _ in pulldown_cmark::Parser::new(data) {};
```

Comment:
This should be fuzzing options too, since default options seem fine, but extended options crash quickly.